### PR TITLE
university-de-dschang.csl

### PR DESCRIPTION
**Pull request by JPRCam**
This is a modified version of the APA.csl.

### CODE REVIEW

**Comment by POBrien333** [comment]
Please go through this and make sure you have everything as requested. Some bits missing. https://github.com/citation-style-language/styles/wiki/Style-Requirements

File: university-de-dschang.csl [hunk 1]
```diff
@@ -0,0 +1,719 @@
...
+    </contributor>
+    <contributor>
+      <name>Sebastian Karcher(APA)</name>
+    </contributor>
+    <contributor>
+      <name> Brenton M. Wiernik(APA)</name>
+      <email>zotero@wiernik.org</email>
+    </contributor>
+    <contributor>
+      <name>John Roettele (APA-modified for University de Dschang)</name>
```
Comment (POBrien333) [inline university-de-dschang.csl]: Put yourself as author, remove all the contributors.
Add <link href="http://www.zotero.org/styles/apa" rel="template"/> after line 7.

File: university-de-dschang.csl [hunk 2]
```diff
@@ -0,0 +1,719 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
+  <info>
+    <title>University de Dschang (APA-modified)</title>
+    <title-short>UniDs</title-short>
```
Comment (POBrien333) [inline university-de-dschang.csl]: remove this title-short.

File: university-de-dschang.csl [hunk 3]
```diff
@@ -0,0 +1,719 @@
...
+    <id>http://www.zotero.org/styles/university-de-dschang</id>
+    <link href="http://www.zotero.org/styles/university-de-dschang" rel="self"/>
+    <author>
```
Comment (POBrien333) [inline university-de-dschang.csl]: Add a link with rel="documentation" to the author guidelines.

File: university-de-dschang.csl [hunk 4]
```diff
@@ -0,0 +1,719 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
```
Comment (POBrien333) [inline university-de-dschang.csl]: Which languages will this style be used for?

File: university-de-dschang.csl (hunk 1 above)
Comment (JPRCam) [reply university-de-dschang.csl]: Okay.

File: university-de-dschang.csl (hunk 3 above)
Comment (JPRCam) [reply university-de-dschang.csl]: What do you mean? This is my first time attempting something like this.

File: university-de-dschang.csl (hunk 2 above)
Comment (JPRCam) [reply university-de-dschang.csl]: done.

File: university-de-dschang.csl (hunk 4 above)
Comment (JPRCam) [reply university-de-dschang.csl]: French and English

File: university-de-dschang.csl (hunk 3 above)
Comment (POBrien333) [reply university-de-dschang.csl]: See points 8 and 10 here: https://github.com/citation-style-language/styles/wiki/Style-Requirements

File: university-de-dschang.csl (hunk 4 above)
Comment (JPRCam) [reply university-de-dschang.csl]: changed "es" to "fr"

**Comment by rmzelle** [comment]
@JPRCam, thanks for submitting a style. Is the style guide for your style online available?

**Comment by JPRCam** [comment]
There is not. It is like APA with only the inline citation dropping the
Author name and adding a space on either side of the :

-Thanks

**Comment by adam3smith** [comment]
We have a strong preference for being able to refer back to the style guide/rules for all styles we host; e.g. if someone in the future claims that there's a mistake in the style, we need to be able to check. Is there a styleguide you could attach here? Or what do you base this style on?

**Pull request closed without merging**